x-pack/legacy/plugins/ml/public/datavisualizer/index_based/route.ts

```diff
@@ -7,19 +7,18 @@
 // @ts-ignore
 import uiRoutes from 'ui/routes';
 // @ts-ignore
-import { checkBasicLicense } from '../license/check_license';
+import { checkBasicLicense } from '../../license/check_license';
 // @ts-ignore
-import { checkGetJobsPrivilege } from '../privilege/check_privilege';
+import { checkGetJobsPrivilege } from '../../privilege/check_privilege';
 // @ts-ignore
-import { loadCurrentIndexPattern, loadCurrentSavedSearch } from '../util/index_utils';
-// @ts-ignore
-import { checkMlNodesAvailable } from '../ml_nodes_check/check_ml_nodes';
+import { loadCurrentIndexPattern, loadCurrentSavedSearch } from '../../util/index_utils';
 // @ts-ignore
+import { checkMlNodesAvailable } from '../../ml_nodes_check/check_ml_nodes';
 import { getDataVisualizerBreadcrumbs } from './breadcrumbs';
 
 const template = `<ml-nav-menu name="datavisualizer" /><ml-data-visualizer />`;
 
-uiRoutes.when('/data_visualizer', {
+uiRoutes.when('/jobs/new_job/datavisualizer', {
```

Comment:
Can you explain why this route is now prefixed with `/jobs/new_job/`? Do we maybe need to consider a redirect to support bookmarks to the old route?

Reply:
The route `/jobs/new_job/datavisualizer` is unchanged - it used to be defined in `datavisualizer_controller.js` for the old AngularJS page. It contains `jobs/new_job` as the page was originally only accessed from the ''Create new job' page, where it lists the types of job wizards available and also a link to the Data Visualizer. 

The directive for the selector page in the top-level folder contains the `datavisualizer` route. `/data_visualizer` was only there temporarily whilst I had links for the old and new data visualizer pages.

Reply:
Thanks for clarifying!